### PR TITLE
Closes #4, Configure Python Only Integration Example for Distance

### LINKED ISSUE #4: Configure Python Only Integration Example for Distances
In the main arkouda repository, `examples/cos_dist.py` demonstrates how to use Arkouda to compute cosine distance. We want to take this one step further and configure an example that will demonstrate how to add this functionality as an external Arkouda python only module.

- Cosine Distance
- Euclidean Distance

### PR DESCRIPTION
**Pull request by Ethan-DeBandi99**
This PR (Closes #4):

This takes the `arkouda/cos_dist.py` from the main Arkouda repository and configures it as an external Arkouda module. I structured the directories as suggested in issue #2 so that we can evaluate if this is what we want to continue with. 

I also added a configuration file, `module_configuration.py`, to provide the commands to path the module so that we can access it from any directory. This is really only necessary if you would configure something locally and want to path it without installing via a package manager. This file will be expanded to handle the server modules as well when we add that version of this module. 

This module is an example using only client side code to implement cosine and euclidean distance for 2 `pdarrays`.

### CODE REVIEW

**Comment by Ethan-DeBandi99** [comment]
Meant to submit this as a draft as it still needs some cleanup, but thought it would be good to get the basic outline pushed up. I am working on the server version as well.

**Comment by Ethan-DeBandi99** [comment]
The current configuration method is only functional if a user runs the export command manually. I am looking into other ways to achieve the desired effect that could be done automatically either via a pipe or script execution.

It is worth noting that writing the export command to a .sh file and then running `source file.sh` appears to achieve the desired effect, so we may be able to do something similar to the tmp config file being created for the server.

**Comment by Ethan-DeBandi99** [comment]
Updated this to utilize pip install. This method is more in line with the server configuration.

**Comment by Ethan-DeBandi99** [comment]
I added testing. base_test.py is copied from `/arkouda/tests/base_test.py`.

File: arkouda_distance/client/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+- `magnitude()` - Compute the magnitude/l2-norm of Arkouda `pdarray`.
+- `cosine()` - Compute the cosine distance of 2 Arkouda `pdarrays`.
+- `euclidean()` - Compute the Euclidean distance of 2 Arkouda `pdarrays`.
+
+## Usage
+
+Arkouda must be installed prior to utilization.
+
+```commandline
+pip install arkouda
```
Comment (mhmerrill) [inline arkouda_distance/client/README.md:17]: should this be `pip install arkouda_distance` ?

Comment (mhmerrill) [reply arkouda_distance/client/README.md:17]: oops I didn't read closely enough.

**Review by mhmerrill — APPROVED** [review]
i am ok with what is here as a start.

**Review by pierce314159 — APPROVED** [review]
Looks good

File: README.md [hunk 2]
```diff
@@ -1,2 +1,64 @@
...
+env =
+    D:ARKOUDA_SERVER_HOST=localhost
+    D:ARKOUDA_SERVER_PORT=5555
+    D:ARKOUDA_RUNNING_MODE=CLASS_SERVER
+    D:ARKOUDA_NUMLOCALES=2
+    D:ARKOUDA_VERBOSE=True
+    D:ARKOUDA_CLIENT_TIMEOUT=0
+    D:ARKOUDA_LOG_LEVEL=DEBUG
+```
+**NOTE** - All test definition naming should be of the format `test_<name>`.
```
[the remark this answers is not in the document]
Comment (Ethan-DeBandi99) [reply README.md]: Yes, this is referring to the methods. Wording has been updated.

**Review by mhmerrill — APPROVED** [review]
looks good.

**Pull request merged**